lib/index.js

```diff
@@ -706,89 +601,68 @@ function _connect(options, callback) {
  *
  * @param {object} options - Options to use.
  * @param {object} options.auth - user and password credentials.
+ * @param {string} options.auth.user - The MongoDB usename.
+ * @param {string} options.auth.password - The MongoDB password.
  * @param {object} options.opts - Options for the MongoClient.
+ * @param {Server} options.url - A mongo connection string.
  * @param {Function} options.callback - A callback function.
- * @param {Server} options.server - The mongo server to connect to.
  *
  * @returns {Promise} The result of the connect.
 */
-async function _loginUser({auth, opts, callback, server}) {
-  const client = new MongoClient(server, {
-    auth,
-    ...opts
-  });
-  return client.connect(callback);
+async function _loginUser({auth, opts, url, callback}) {
```

Comment:
Should this be `async`? It takes a callback ... and doesn't seem to be used in some backwards compatible way, it only seems to use the `callback` version. Maybe this is ok as is, just asking.

Reply:
I believe when I started I had intended to make this module `async` entirely. As written this is not async so I should probably remove the `async` tag for now. I do think in the future this module should get an over haul to `async` `await` code.

Reply:
Doesn't look like it should have async here.  We'll fix up everything at once to be async/await.

Reply:
fixed here: https://github.com/digitalbazaar/bedrock-mongodb/pull/52/commits/056abb3862e88f5d2b28dddf68591cf6124db0ba